module/data/other/roll-options.mjs

```diff
@@ -250,6 +250,13 @@ export default class EdRollOptions extends foundry.abstract.DataModel {
           step: 1,
           integer: true,
         } ),
+        pointsRequired: new foundry.data.fields.BooleanField( {
```

Comment:
Why do we need that? The roll doesn't care if its required or not. That check should be done before rolling.

Reply:
the user can reduce the devotion points in the prompt